### PR TITLE
[Engineering] setting up database

### PR DESCRIPTION
**Pull request by henrytomsf**
1. database set up in aws
2. auth to db
3. read/write to db

### CODE REVIEW

File: database/dbQuery.py [hunk 1]
```diff
@@ -19,4 +51,23 @@ def getQuery(dc,
     query = 'SELECT version()'
     results = getQuery(dc, query)
     print(results)
+
+    # test 2
+    query = '''
```
Comment (vincentpham1991) [inline database/dbQuery.py:57]: shouldn't you put these in unit tests?

File: database/dbQuery.py [hunk 2]
```diff
@@ -19,4 +51,23 @@ def getQuery(dc,
...
+    SELECT * FROM test_table
+    '''
+    results = getQuery(dc, query)
+    print(results)
+
+    # test 3
+    table = 'test_table'
+    dataTuple = [(6, 'vinny', 'vinny@gmail.com'),
```
Comment (vincentpham1991) [inline database/dbQuery.py:65]: no

**Review by vincentpham1991 — CHANGES_REQUESTED** [review]

File: database/dbQuery.py (hunk 1 above)
Comment (henrytomsf) [reply database/dbQuery.py:57]: added TODO for you 🎉

File: database/dbQuery.py (hunk 2 above)
Comment (henrytomsf) [reply database/dbQuery.py:65]: yes

**Review by vincentpham1991 — APPROVED** [review]
Fine!

**Pull request merged**